akka-http/src/main/mima-filters/10.1.1.backwards.excludes

```diff
@@ -0,0 +1,4 @@
+# Only show up on scala 2.11:
```

Comment:
I guess these ones are ok because 1. PredefinedToResponseMarshallers/LowPriorityToResponseMarshallerImplicits are de-facto `@DoNotInherit` and 2. Java users should probably use the Scala 2.12 version anyway?